andela/fargo-ah-client
# #160238582 Header card component

#### What does this PR do?

Header card component

#### Description of Task to be completed?
  - Write unit tests
  - Add semantic-ui CDN
  - Add semantic-ui-react
  - Create the header card component

#### How should this be manually tested?
In ComandLine:
- `git clone https://github.com/andela/fargo-ah-client.git Authors-Haven-Client`
- `npm install`
- `npm run start` the start script
- Go to '/ ' path to check the home route

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#160238582](https://www.pivotaltracker.com/story/show/160238582)

#### Screenshots

<img width="1498" alt="screen shot 2018-09-04 at 4 56 18 pm" src="https://user-images.githubusercontent.com/38701151/45043033-0f04ef80-b064-11e8-8e8f-63c38b60a3c3.png">
